Classes/ContentObject/FileContentObject.php

```diff
@@ -34,7 +33,6 @@ public function render($conf = [])
         $theValue = '';
         $file = isset($conf['file.']) ? $this->cObj->stdWrap($conf['file'], $conf['file.']) : $conf['file'];
         try {
-            $file = GeneralUtility::makeInstance(FilePathSanitizer::class)->sanitize($file, true);
```

Comment:
I suggest to — instead of removing this code, to move it into an else part three lines below:
if (str_starts_with(EXT:...) {
} else {
$file = GeneralUtility::makeInstance(FilePathSanitizer::class)->sanitize($file, true);
}